src/hmem_cuda.c

```diff
@@ -839,6 +839,11 @@ int cuda_get_base_addr(const void *ptr, void **base, size_t *size)
 	return -FI_ENOSYS;
 }
 
+enum cuda_xfer_setting cuda_get_xfer_setting(void)
+{
+	return CUDA_XFER_UNSPECIFIED;
```

Comment:
For a non cuda build, we should return not allowed.

Reply:
CUDA_XFER_DISABLED

Reply:
Done.